File: backend/src/main/java/com/corusoft/ticketmanager/common/dtos/FieldErrorDTO.java
package com.corusoft.ticketmanager.common.dtos;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class FieldErrorDTO {
    /**
     * Nombre del campo que provoca el error
     */
    private String fieldName;

    /**
     * Mensaje detallado con el problema
     */
    private String message;
}

File: mobile/app/src/main/java/com/corusoft/ticketmanager/backend/dtos/common/FieldErrorDTO.kt
package com.corusoft.ticketmanager.backend.dtos.common

data class FieldErrorDTO(
    val fieldName: String?,
    val message: String?
) {
    constructor() : this(null, null)

    override fun toString(): String {
        return "Error en $fieldName: $message"
    }
}
